docs/templating/templates/view_matcher_reference.md

```diff
@@ -32,6 +32,8 @@ You can use the following matchers to [match content views](template_configurati
 
 !!! tip
 
+    Each matcher has a scalar value or an array of scalar values. When an array is passed, it matches on one of its values.
```

Comment:
Already there in 2.5: https://doc.ibexa.co/en/2.5/guide/content_rendering/#matcher-value